modules/virt-about-cluster-checkup-framework.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
```

Comment:
I think this needs to be updated to the `virt-running-cluster-checkups` assembly